src/mongo/NoStat.php

```diff
@@ -0,0 +1,64 @@
+<?php
```

Comment:
This is just moved into its own file and linting issues cleaned up.  There's nothing "new" here.